torch/_dynamo/variables/constant.py

```diff
@@ -78,6 +82,12 @@ def call_method(
                 items=self.unpack_var_sequence(tx), source=self.source, **options
             ).call_method(tx, name, args, kwargs)
 
+        if any([isinstance(x, DynamicShapeVariable) for x in args]):
+            if name == "__add__":
+                assert len(args) == 1
+                return args[0].call_method(tx, name, [self], {})
```

Comment:
Relying on commutativity 🤔

Reply:
Safe for add, right?

Reply:
I'm mostly afraid someone will copy this code for div and oops

Reply:
Perhaps a comment?

Reply:
Well, the more correct thing to do is swizzle this into `__radd__` and do the reverse call